src/workers/parser/getSymbols.js

```diff
@@ -291,7 +294,8 @@ function extractSymbols(sourceId): SymbolDeclarations {
     imports: [],
     literals: [],
     hasJsx: false,
-    hasTypes: false
+    hasTypes: false,
+    loading: false
```

Comment:
we can no longer check if a property exists w/ flow... with that said, we might want to try another option here

Reply:
👍 interesting note, just saw that in 0.68 release notes.